### DISCUSSION TITLE
Can I use match without a shared Apple developer account?

### DISCUSSION THREAD

**Discussion by bviebahn** [fastlane/fastlane#20730] [Q&A]
In the docs for match it says:
> Optional: Create a new, shared Apple Developer Portal account, something like office@company.com, that will be shared across your team from now on (for more information visit [codesigning.guide](https://codesigning.guide/))

On codesigning.guide I can find this:
> The basic requirement is to have one code signing identity shared across your team. The easiest way to do that is to create a new Apple ID for the team (e.g. ios-dev@company.com) and use that from now on.

Now I'm a bit confused, is this shared account optional or is it a requirement to use this form of codesigning? If we can also use multiple accounts, what are the disadvantages of that?

Thanks for your help

**Comment by tarbaiev-smg** [comment]
Yes you can. You just need to specify your username and password (or an API key) during the Match run instead of setting it in the Matchfile.
The main disadvantage is inconvenience. But on the flip side you get better access control as you can limit Admin rights to certain people in your team.
